oshmem/proc/proc.h

```diff
@@ -38,8 +38,8 @@ struct oshmem_group_t;
 #define OSHMEM_PE_INVALID   (-1)
 
 struct oshmem_proc_data_t {
```

Comment:
@ggouaillardet could you add comment for this struct that says that this struct should meet padding size in ompi_proc_t as I asked. Thanks.